stockfish/models.py

```diff
@@ -36,7 +36,7 @@ def __init__(
             path, universal_newlines=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE
         )
 
-        self._stockfish_major_version: int = int(self._read_line().split(" ")[1])
+        self._stockfish_major_version: float = float(self._read_line().split(" ")[1])
```

Comment:
`_stockfish_major_version` should stay `int`

If stockfish version is `14.1`, this variable should contain `int` value `14`

Reply:
please add test for this case

Reply:
Fixed

Reply:
Well done